Nim/Program.cs

```diff
@@ -0,0 +1,441 @@
+                foreach (string menuLine in menuLines) {
+                    if (menuLine.Contains("Please select an option")) {
+                        Console.ForegroundColor = ConsoleColor.Magenta;
+                        
+                    } else if (menuLine.Contains("<<  | [A] - Nim Game (basic) |  >>")) {
+                        Console.ForegroundColor = ConsoleColor.Yellow;
+                        
+                    } else if (menuLine.Contains("<<  | [B] - Nim Game (fun)   |  >>")) {
+                        Console.ForegroundColor = ConsoleColor.Green;
+
+                    } else if (menuLine.Contains("¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨") ||
+                               menuLine.Contains("__________________________") || 
+                               menuLine.Contains($"| Wins: {_winCountBasicMenu}     Loses: {_lostCountBasicMenu}   |") ||
+                               menuLine.Contains($"| Wins: {_winCountFunMenu}     Loses: {_lostCountFunMenu}   |")) 
+                    {
+                        Console.ResetColor();
+                    } else {   
+                        Console.ForegroundColor = ConsoleColor.Red; // ASCII ART
+                    }
+                    Console.WriteLine(menuLine); //Print the newly coloured Menu.
+                } 
+                Console.ResetColor();
+                                                   //use ! to suppress warning
+                string menuSelection = Console.ReadLine()!.ToUpper(); //Read menuLine. Console.Readline returns string, change to var?
+                switch (menuSelection) {
+                    case "A": BasicNim(); break;
+                    case "B": FunNim(); break;
+                    case "Q":
+                        exitProgram = true;
+                        break;
+                    default:
+                        Console.WriteLine("Invalid choice! Please select one of the appropriate options.");
+                        break;
+                }
+            }
+        }
+        static void BasicNim()
+        {
+            bool backToMenu = false;
+            while (!backToMenu) {
+                Console.WriteLine();
+                Console.WriteLine("You're inside of Nim Game (basic)!");
+                Console.WriteLine("Press 'C' to Confirm. Press 'Q' to go to Main Menu.");
+                Console.WriteLine($@"You've won {_winCountBasicMenu} times. You've lost {_lostCountBasicMenu} times.");
+
+                string menuSelection = Console.ReadLine()!.ToUpper();
+                if (menuSelection == "C") {
+                    //If instead of switch for fun
+                    Console.WriteLine();
+                    Console.WriteLine("Welcome to Nim!");
+                    int matchesCount = 24;
+                    int playerMove = 0;
+                    bool playerTurn = false;
+                    bool aiTurn = false;
+                    bool gameOver = false;
+                    Random ai = new Random();
```

Comment:
Consider using `Random.Shared`. I like that you define all game parameters right away! :)